pkgs/by-name/cr/creek/build.zig.zon.nix

```diff
@@ -0,0 +1,111 @@
+# generated by zon2nix (https://github.com/Cloudef/zig2nix)
+{
+  lib,
+  linkFarm,
+  fetchurl,
+  fetchgit,
+  runCommandLocal,
+  zig,
+  name ? "zig-packages",
+}:
+with builtins;
+with lib;
```

Comment:
```suggestion
```
Use inherit or add `lib.` / `builtins.` below.

Reply:
I realize the file is generated. The generator must be updated to not generate `with` statements.